Bottom: 595f77f805a6dbf7ff38934380961382a5ae1eff
Top:    9fa3076191cf62962b91660fa47c9298fd492afb
Author: student <student@ReliableEmbeddedSystems.com>
Date:   2020-08-26 11:15:29 +0300

make bsp tool dunfell compatible

Signed-off-by: student <student@ReliableEmbeddedSystems.com>


---

diff --git a/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf b/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
index b18c480656..5a914ef1ed 100644
--- a/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
+++ b/scripts/lib/bsp/substrate/target/arch/common/conf/layer.conf
@@ -9,4 +9,4 @@ BBFILE_COLLECTIONS += "{{=machine}}"
 BBFILE_PATTERN_{{=machine}} = "^${LAYERDIR}/"
 BBFILE_PRIORITY_{{=machine}} = "6"
 
-LAYERSERIES_COMPAT_{{=machine}} = "sumo thud warrior zeus"
+LAYERSERIES_COMPAT_{{=machine}} = "sumo thud warrior zeus dunfell"
